glide.yaml

```diff
@@ -16,6 +16,8 @@ testImport:
 - package: golang.org/x/tools
   subpackages:
   - cover
-- package: github.com/golang/lint
+- package: golang.org/x/lint
```

Comment:
i'm not clear on why this change is necessary, especially if we're under the hood doing the rewriting through glide

Reply:
We ran into this in YARPC and ThriftRW: With golang/lint@c363707,
golint imports another package from its repo using the golang.org/x/lint
import path.

If you use github.com/golang/lint, and try to build 
github.com/golang/lint/golint, it will fail because there's nothing
under vendor/golang.org/x/lint, which is what it's importing.